docs/guides/Versioning.rst

```diff
@@ -1,105 +1,107 @@
+.. _versioning:
+
 Versioning
 ##########
 
-We use `semantic-release <https://github.com/semantic-release/semantic-release>`_ to automatically determine the next version of a project **and** publish a release in the GitHub project as well as release it to our npm organization ``@NordicSemiconductor``.
+`Semantic-release <https://github.com/semantic-release/semantic-release>`_ is used to automatically determine the next version of the project and publish a release in the GitHub project as well as release it to the ``@NordicSemiconductor`` npm organization .
```

Comment:
```suggestion
`semantic-release <https://github.com/semantic-release/semantic-release>`_ is used to automatically determine the next version of the project and publish a release in the GitHub project as well as release it to the ``@NordicSemiconductor`` npm organization .
```

The project is named `semantic-release` with lower-case `s`: https://github.com/semantic-release/semantic-release